tests/gl_draw_elements.c

```diff
@@ -0,0 +1,53 @@
+#include <GL/gl.h>
+#include <GL/glut.h>
+
+typedef struct Vertex {
+    short           x, y, z;
+    unsigned char   c_r, c_g, c_b, c_a;
+    short           u,v;
+} Vertex;
+
+Vertex triangle[3] = {
+    0, 0, 0,   0, 255, 255, 255, 0, 0,
+    1, 1, 0, 255,   0, 255, 255, 0, 0,
+    1, 0, 0, 255, 255,   0, 255, 0, 0
```

Comment:
Above, struct Vertex defines x, y, z, u and v as `short`s, but it looks like this array fills them in as if they were `unsigned char`s. Is that intentional for this test to provide this kind of a mismatched input array?